caktus/etlman
# Schedule pipeline

## Summary
Allows users to schedule pipeline runtimes.

## Test
**Note**: This PR includes migration changes
1. Run all tests and ensure they pass
2. Apply the migrations and create a new pipeline (including the new runtime schedule). After clicking `Save pipeline`, you should be redirected to the `schedule` page
3. Log into the Django admin and ensure that the Pipeline Schedule was created successfully
4. Edit the existing schedule for a pipeline (button is the 'clock' located in the pipeline list), ensuring that the changes took effect in the Django admin
